beamer/health/check.py

```diff
@@ -192,27 +210,54 @@ def fetch_events() -> ChainEventMap:
     return cast(ChainEventMap, events)
 
 
-def make_web3(rpc: str) -> Web3:
-    web3 = Web3(Web3.HTTPProvider(rpc, request_kwargs=dict(timeout=5)))
-    web3.middleware_onion.inject(geth_poa_middleware, layer=0)
-    return web3
+def get_transfer_token_symbol(transfer: Transfer, token_deployments: TokenMap) -> str | None:
+    for token_symbol, deployments in token_deployments.items():
+        if any(
+            transfer["created"].source_token_address.lower() == address.lower()
+            and transfer["created"].chain_id == int(chain_id)
+            for chain_id, address in deployments
+        ):
+            return token_symbol
+
+    return None
+
+
+def get_transfer_value_formatted(
+    request: RequestCreated, token_deployments: TokenMap, token_details: TokenDetailsMap
+) -> str:
+    token_source_chain = request.chain_id
+    token_source_address = request.source_token_address
+    token = None
+    transfer_value = "0"
+
+    for token_symbol, deployments in token_deployments.items():
+        if deployments.count([str(token_source_chain), token_source_address]):
```

Comment:
Why do we construct a list as the argument to `count`?

Reply:
Ah, I see. Then I suggest to pull the list construction out of the loop, since it is loop invariant so you create the list only once. And then you could use the more Python `x in sequence` expression like
```python
    token = [str(token_source_chain), token_source_address]
    for token_symbol, deployments in token_deployments.items():
        if token in deployments:
            ...
```